content/docs/04.5-devx/05-app-profile/02-container-deployment.md

```diff
@@ -19,21 +19,34 @@ To add a container tier to Palette Dev Engine App Profile,
 
```

Comment:
We need to clean up the intro. We need to explain why someone would benefit from this. Take a look at how we did the other How-to's.

![CleanShot 2022-11-28 at 07 18 35](https://user-images.githubusercontent.com/29551334/204300312-eb9c3330-8a9b-46be-a720-8ce781d51d40.png)

Reply:
done!